server/src/infra/repositories/person.repository.ts

```diff
@@ -50,7 +50,15 @@ export class PersonRepository implements IPersonRepository {
     return people.length;
   }
 
-  getAll(userId: string, options?: PersonSearchOptions): Promise<PersonEntity[]> {
+  getAll(): Promise<PersonEntity[]> {
+    return this.personRepository.find();
+  }
+
+  getAllWithoutThumbnail(): Promise<PersonEntity[]> {
+    return this.personRepository.createQueryBuilder('person').where("person.thumbnailPath = ''").getMany();
```

Comment:
I think this would work as well? We prefer to avoid the query builder where possible.

```suggestion
    return this.personRepository.findBy({ thumbnailPath: "''" });
```